src/test/java/hw2/ComputerTest.java

```diff
@@ -0,0 +1,34 @@
+package hw2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ComputerTest {
+    Computer com1 = new Computer("Hp", 15);
+    Computer com2 = new Computer("Asus", 17, 8);
+    Computer com3 = new Computer("Hp", 17, "laptop");
+
+    @Test
+    @DisplayName("Check com1")
+    public void Comp1() {
```

Comment:
Please change method names according to Java Naming Conventions - lowerCamelCase